teuthology/test/test_scrape.py

```diff
@@ -165,3 +175,31 @@ def test_scraper(self):
         assert os.path.exists(os.path.join(d.path, "scrape.log"))
 
         shutil.rmtree(d.path)
+
+    def test_gzip_backtrace_decode(self):
+        import gzip, os, glob
```

Comment:
Could you please move imports to the top of the test module.
First of all we need to know about all dependencies as early as possible, from other side, the 'os' is already imported.